src/frontend/src/forms/PurchaseOrderForms.tsx

```diff
@@ -298,6 +301,25 @@ function LineItemFormRow({
     }
   });
 
+  const [expiryDateOpen, expiryDateHandlers] = useDisclosure(false, {
+    onOpen: () => {
+      // check the default part expiry. Assume expiry is relative to today
+      const defaultExpiry = record.part_detail?.default_expiry;
```

Comment:
I am assuming the number of days specified in `default_expiry` should be relative to the receive date.

I'm not well-versed in formatting javascript dates... but coming from Python this doesn't feel great. Open to suggestion here if there is a better way

Reply:
Could it be as simple as adding N days to the current date, right?

Reply:
Yes... that is what I'm doing... but the formatting is what gets so verbose (lines 311-314). Does the form require me to format it in `"YYYY-MM-DD"`?

Reply:
We already have `dayjs` dependency in the frontend code, so you could do something like:

```javascript
// Generate a date 10 days in the future
const futureDate = dayjs().add(10, 'day').format('YYYY-MM-DD');
```

Reply:
I like that much better. Thanks for the tip. Updated